### ISSUE TITLE
[Feature request] Copy code block to clipboard

### ISSUE THREAD

**Issue by C0rn3j** [wekan/wekan#5149]
I would appreciate it if WeKan had the ability to copy code blocks easily, similarly to how [GitHub does it](https://stackoverflow.com/a/67478530).

I have saved workflows in WeKan with lots of code blocks and having to manually edit each checklist item to copy the code out is tedious.

On a similar note, is there a way to get code highlight? I am currently stuck on 6.09, so apologies if that is already possible on a newer version.

**Comment by xet7** [comment]
@C0rn3j 

I added copy code block to clipboard.

There is no code hightlight currently.

**Comment by mfilser** [comment]
@xet7
If i remember right there is already code in the Utils class to copy text to the clipboard...

**Comment by xet7** [comment]
@mfilser 

Yes, that is for copying all text from card details when editing card details.

This new code is for copying code blocks when viewing card details. That copy button is visible after opening card, but not anymore after editing card details, I did not yet figure out how to keep it visible.

**Comment by xet7** [comment]
@mfilser 

Also, that copy to clipboard text is not yet translated.

**Comment by xet7** [comment]
@C0rn3j 

You can test it with this Docker image:
https://github.com/wekan/wekan/blob/main/docker-compose.yml#L126

**Comment by C0rn3j** [comment]
Sorry, got to testing it only now.

It seems to work very sporadically (only a handful of my checklist items have it) and upon opening to edit them, they lose it.

This is with latest 7.44 on Docker on Chromium.

[Screencast_20240501_135332.webm](https://github.com/wekan/wekan/assets/1641362/7d039ff9-9fe9-40e4-8b3b-5d43cf3a8daa)
